endoreg_db/serializers/video/segmentation.py

```
@@ -0,0 +1,242 @@
+from pathlib import Path
+from rest_framework import serializers
+from ...models import VideoFile, LabelVideoSegment # Added VideoPredictionMeta
```

Ruff (0.12.2)

`...models.LabelVideoSegment` imported but unused

Remove unused import: `...models.LabelVideoSegment`

(F401)